catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/CreateRequestImpl.java

```diff
@@ -60,8 +65,19 @@ public CreateRequestImpl(List<Metacard> metacards) {
      *            the properties
      */
     public CreateRequestImpl(List<Metacard> metacards, Map<String, Serializable> properties) {
+        this(metacards, properties, new HashSet<>());
+    }
+
+    public CreateRequestImpl(List<Metacard> metacards, Map<String, Serializable> properties,
+            Set<String> destinations) {
         super(properties);
         this.metacards = metacards;
+        this.destinations = destinations;
```

Comment:
Do you want to check for null destinations being passed in and set to new HashSet<>() instead so that the behavior is consistent and this field is never null?